src/models/Settings.js

```diff
@@ -166,6 +166,9 @@ export default Model.extend({
 
     //PIV
     piv: ['object', false, {}],
+
+    //Email verify callback
+    stateTokenExternalId: 'string'
```

Comment:
Internal use only? or expose this config in README?

Reply:
I think we will leave this undocumented for now until we update our samples and other documentation. There may be additional changes to this flow.